src/main/kotlin/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,41 @@
+package lotto.model
+
+import lotto.util.Constant
+
+class Lotto(val numbers: Set<LottoNumber>) {
+    init {
+        require(numbers.size == Constant.LOTTO_LEN)
+    }
+
+    fun matchCount(winningNumber: WinningNumber): Int {
+        return winningNumber
+            .getWinning()
+            .numbers
+            .intersect(numbers)
+            .size
+    }
+
+    fun matchBonusNumber(winningNumber: WinningNumber): Boolean {
+        return numbers.contains(winningNumber.getBonusNumber())
+    }
+
+    fun findRanking(winningNumber: WinningNumber): LottoPrize {
+        var rank =
+            LottoPrize.entries.find {
+                it.getMatchNumbers() == matchCount(winningNumber)
+            } ?: LottoPrize.BOOM
+        if (checkSecond(rank, matchBonusNumber(winningNumber))) {
+            rank = LottoPrize.SECOND
+        }
+        return rank
+    }
+
+    fun contains(number: LottoNumber): Boolean {
+        return numbers.contains(number)
+    }
```

Comment:
객체에게 메시지를 던진 좋은 예시네요!

아래 처럼 연산자를 활용해볼 수도 있답니다 :)
```suggestion
    fun contains(number: LottoNumber): Boolean {
        return number in numbers
    }
```

또, `operator fun contains(~)~~` 가 무엇인지도 알아보시면 좋겠어요 :)